gravitee-apim-cypress/cypress/integration/rest-api/apis/api-tests.ts

```diff
@@ -13,8 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { ADMIN_USER, API_PUBLISHER_USER, LOW_PERMISSION_USER } from 'fakers/users/users';
-import { ApiFakers } from 'fakers/apis';
+import { ADMIN_USER, API_PUBLISHER_USER, LOW_PERMISSION_USER } from 'fixtures/fakeData/users/users';
```

Comment:
I am not a big fan of `*-tests.ts` naming.
The convention is `*.spec.ts`

but you can keep your new files with `*-tests.ts` so it will be easier to know which file to refactor

Reply:
This was exactly the intention behind it. I also prefer the *.spec.ts format but I wanted to make them look differently to identify them later more easily for the refactoring.